docs/compatibility.md

```diff
@@ -457,12 +462,6 @@ The following regular expression patterns are not yet supported on the GPU and w
 - Empty groups: `()`
 - `regexp_replace` does not support back-references
 
-The following regular expression patterns are known to potentially produce different results on the GPU
-vs the CPU.
-
-- Word and non-word boundaries, `\b` and `\B`
```

Comment:
We had two sections listing known edge cases, so I consolidated them by moving this content.